### PR TITLE
[ART-7056] deprecate placeholder bug

### PR DESCRIPTION
**Pull request by Ximinhan**
remove the deprecated use of placeholder bug

### CODE REVIEW

File: elliott/elliottlib/cli/create_textonly_cli.py [hunk 1]
```diff
@@ -108,10 +108,6 @@ def create_textonly(runtime, errata_type, date, assigned_to, manager, package_ow
         erratum.commit()
         green_prefix("Created new text only advisory: ")
         click.echo(str(erratum))
-        bug = bug_tracker.create_textonly(bug_title, bug_description)
```
Comment (vfreex) [inline elliott/elliottlib/cli/create_textonly_cli.py:111]: This line shouldn't be removed.

**Review by locriandev — COMMENTED** [review]
/lgtm

File: elliott/elliottlib/cli/create_textonly_cli.py (hunk 1 above)
Comment (Ximinhan) [reply elliott/elliottlib/cli/create_textonly_cli.py:111]: text-only advisory is also RHOSE product which doesn't need a placeholder bug

Comment (vfreex) [reply elliott/elliottlib/cli/create_textonly_cli.py:111]: ack

**Pull request merged**